cmd/gin-repod/repo.go

```diff
@@ -749,27 +749,29 @@ func (s *Server) listRepoCollaborators(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	type collaborator struct {
+		User        string `json:"User"`
```

Comment:
The `json:User` part as well es the `json:AccessLevel` part are not really required, but I guess they don't hurt much either.